src/ts/ConvivaAnalytics.ts

```diff
@@ -346,15 +349,18 @@ export class ConvivaAnalytics {
    *  - encodedFrameRate (unused)
    */
   private internalInitializeSession() {
-    // initialize PlayerStateManager
-    this.playerStateManager = this.client.getPlayerStateManager();
-    this.playerStateManager.setPlayerType('Bitmovin Player');
-    this.playerStateManager.setPlayerVersion(this.player.version);
-
     this.buildContentMetadata();
 
     // Create a Conviva monitoring session.
-    this.sessionKey = this.client.createSession(this.contentMetadataBuilder.build()); // this will make the initial request
+    this.convivaVideoAnalytics = Conviva.Analytics.buildVideoAnalytics();
+    this.convivaVideoAnalytics.reportPlaybackRequested(this.contentMetadataBuilder.build());
+    this.sessionKey = this.convivaVideoAnalytics.getSessionId();
+    this.convivaVideoAnalytics.setCallback(() => {
```

Comment:
According to Conviva docs this fires every one 1 and this is the way they recommend implementing it.

From Conviva Docs:
'There are certain metrics which do not have events, hence Conviva provides a callback to update these values at a certain frequency. Once player has been initialized, implement an callback function for the player so that the VideoAnalytics can get continuous measures of the player such as, play head time, buffer length, and rendered framerate, and other measurements.'

Reply:
I see, thanks 👍